### ISSUE TITLE
when starting with "--geometry=NNNNxNNN" "window-scale" is always "1"

### ISSUE THREAD

**Issue by xvim64** [mpv-player/mpv#9066]
### Important Information

- mpv version : mpv-x86_64-20210725-git-e2109b6.7z
- Windows Version : Windows 10 20H2

### Reproduction steps

![mpv-bug-window-scale](https://user-images.githubusercontent.com/83569599/127592223-4c9cf097-0340-40e7-92cc-e9a276ecb861.gif)

```
1.
1280x720.mp4

input.conf----------
8 set window-scale 1
9 set window-scale 0.5
--------------------


2.
mpv.exe --geometry=1600x900 1280x720.mp4

or

mpv.conf------------
geometry=1600x900
--------------------

mpv.exe 1280x720.mp4


3.
press key 8
```

### Expected behavior

"--geometry=1600x900" makes "windows-scale" "1.25"  (1.25 == 1600/1280)

so "set windows-scale 1" changes mpv screen size 1280x720.

### Actual behavior

The "window-scale" of "--geometry=1600x900" is "1".

There is no mpv screen size change.

**Comment by Dudemanguy** [comment]
Yeah this is unintuitive behavior but currently expected. Basically, `window-scale` is merely and option and not a property or command. By default it is 1 and it does not change if you change the window size/geometry/etc. options. The property that actually keeps track of the window scale at all times is `current-window-scale`. Since the window-scale option is already 1, setting it to 1 again doesn't do anything regardless of the actual size of the window.

For reference, here's an old PR that would make `current-window-scale` writable: https://github.com/mpv-player/mpv/pull/8247

**Comment by Dudemanguy** [comment]
Revisiting this now, I think I would prefer just changing `window-scale` to a property and removing it as an option. I'll probably make that a PR tomorrow.

**Comment by hooke007** [comment]
related issue https://github.com/mpv-player/mpv/issues/7770

**Comment by Dudemanguy** [comment]
Fixed by #8247.
